shared/js/initApp.js

```diff
@@ -291,6 +297,11 @@ function initApp() {
       appOptions.hideSource = true;
       appOptions.callouts = [];
     }
+  } else if (appOptions.isLegacyShare) {
```

Comment:
doesn't need to be fixed now, but I think this conditional is getting a bit on the too-branchy-to-read side